website/views/shared/header/menu.jade

```diff
@@ -99,14 +99,14 @@ nav.toolbar(ng-controller='MenuCtrl')
               a(ui-sref='options.profile.achievements')=env.t('achievs')
             li
               a(ui-sref='options.profile.profile')=env.t('profile')
-      li.toolbar-button-dropdown
+      li.toolbar-button-dropdown(dropdown)
         a(ui-sref='options.social.inbox', ng-if='user.inbox.newMessages')
           span.badge.badge-danger {{user.inbox.newMessages}}
-        a(ui-sref='options.social.tavern', data-close-menu)
+        a(ui-sref='options.social.tavern')
           span=env.t('social')
-        a(ng-class='{active: _expandedMenu.menu === "social"}', data-expand-menu, menu='social')
+        a(dropdown-toggle)
           span &#9776;
-        div(ng-show='_expandedMenu.menu === "social"', data-close-menu)
```

Comment:
It looks like there are references in the close-menu and expand-menu directives (unsurprisingly) and the tests for those directives, and I missed a couple of references in this file on lines [292](https://github.com/HabitRPG/habitica/pull/8657/files/19b9be27a55a6f69c88d538f4d560be298161e4a#diff-0d726286f04628feed0f89ff02f1078bR292) and 296. I'll remove the directives and tests, I guess?

What happens if the `ul.toolbar-bailey` on lines 292/296 have the `inactive` class? With this PR, they never will, but I don't know how to test that or how to see what they do.